StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/FinancialConnectionsAPIClient.swift

```diff
@@ -199,6 +199,9 @@ extension STPAPIClient: FinancialConnectionsAPIClient {
             }(),
             "locale": Locale.current.toLanguageTag(),
         ]
+        if let publishableKey {
+            parameters["key"] = publishableKey
```

Comment:
Will look into whether or not this is needed before merging. It could already be handled here: https://github.com/stripe/stripe-ios/blob/e9b3c5bb/StripeCore/StripeCore/Source/API%20Bindings/STPAPIClient.swift#L213-L223